state/remote/state.go

```diff
@@ -113,7 +123,12 @@ func (s *State) refreshState() error {
 	s.lineage = stateFile.Lineage
 	s.serial = stateFile.Serial
 	s.state = stateFile.State
-	s.readState = s.state.DeepCopy() // our states must be separate instances so we can track changes
+
+	// Properties from the remote must be separate so we can
```

Comment:
A comment covering this might be useful (or perhaps) moved to the type declaration, to make it clear why we have both say, `lineage` and `readLineage`

Reply:
👍 I'll just update it in both places.

Reply:
Yea, I'd say erring on the side of adding more context in both places wouldn't hurt but LMK if just the type declaration is preferred.